WordPress/Classes/ViewRelated/Notifications/FormattableContent/NotificationContentFactory.swift

```diff
@@ -29,12 +29,12 @@ class NotificationContentFactory: FormattableContentFactory {
     }
 
     private static func rangesFrom(_ rawBlock: [String: AnyObject]) -> [FormattableContentRange] {
-        let rawRanges   = rawBlock[Constants.Ranges] as? [[String: AnyObject]]
```

Comment:
I removed a superfluous space here. It seems like SwiftLint should catch that.

Reply:
Thank you for fixing those details! :)